react-backend/shared/controllers/feedback.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+const cbAdmin = require('../models/cb-admin');
+
+const route = ('/',
```

Comment:
A function...

Your `x` is just a bunch of numbers wrapped in parens...

Reply:
The value of `x` in my example is `3`. I was pointing out that assigning `(x, y, z)` to a variable is redundant, because `x` and `y` are ignored. You are assigning `('/', () => {...})` to `route`.

Reply:
Lol, oh yeah, I didn't realise I was doing that.